deployments/examples/ocis_wopi/docker-compose.yml

```diff
@@ -133,22 +129,20 @@ services:
       # share the registry with the ocis container
       MICRO_REGISTRY_ADDRESS: ocis:9233
     volumes:
-      - ./config/ocis-appprovider-collabora/entrypoint-override.sh:/entrypoint-override.sh
       - ocis-config:/etc/ocis
     logging:
       driver: ${LOG_DRIVER:-local}
     restart: always
     depends_on:
-      - ocis
+      ocis:
+        condition: service_started
+      collabora:
+        condition: service_healthy
```

Comment:
https://github.com/owncloud/ocis/pull/8144#discussion_r1444196372

> This is only part of V2, right? V3 https://docs.docker.com/compose/compose-file/compose-file-v3/#depends_on doesn't mention it.

Reply:
Ahh, actually the GO version of docker-compose seems to have it!??? https://docs.docker.com/compose/compose-file/05-services/#long-syntax-1

Reply:
local tests show this is working .... :shrug:

Reply:
running docker  26.1.0